Sources/MapboxDirections/Intersection.swift

```diff
@@ -72,6 +84,18 @@ public struct Intersection {
      If road class information is unavailable, this property is set to `nil`.
      */
     public let outletRoadClasses: RoadClasses?
+
+    public let tunnelName: String?
```

Comment:
Shall we add more descriptive comments to newly added properties?

Reply:
Added some comments across the new properties.